### ISSUE TITLE
Install error on Jessie "dpkg: error processing package webradio-py (--install):  subprocess installed post-installation script returned error exit status 2"

### ISSUE THREAD

**Issue by braselectron** [Acer54/Webradio_v2#19]
Dear developer,

I am testing the built: 0.3.7 on Raspberry Pi 3 Model B version 1.2 running Jessie.

What I did:

`wget https://github.com/Acer54/Webradio_v2/releases/download/0.3.7/webradio-py_0.3.7_armhf.deb`

then to install (using gdebi):

`sudo gdebi webradio-py_0.3.7_armhf.deb`

I did opt to install eyeD3 and not to auto-start at boot and also to keep cursor on screen.

I got the following output:

`~/Downloads $ sudo gdebi webradio-py_0.3.7_armhf.deb 
Reading package lists... Done
Building dependency tree        
Reading state information... Done
Building data structures... Done 
Building data structures... Done 
Requires the installation of the following packages: libadplug-2.2.1-0 libao-common libao4 libbinio1ldbl libdnet libiso9660-8 libmail-sendmail-perl libmpcdec6 libmpdclient2 libmysqlclient18 libnfs4 libqt4-declarative libqt4-designer libqt4-help libqt4-script libqt4-scripttools libqt4-sql libqt4-sql-mysql libqt4-svg libqt4-test libqtassistantclient4 libresid-builder0c2a libroar2 libsidplay2 libsidutils0 libslp1 libsoxr0 libsys-hostname-long-perl libupnp6 libzzip-0-13 mpc mpd mysql-common po-debconf python-httplib2 python-lxml python-mpd python-mpdclient python-qt4 python-sip python-uritemplate 

Webradio Frontend for MPD
 Webradio.py is a Python script which is designed to be a full-screen or windowed Frontend for MPD (Music-Player-Daemon). It allows to manage webradio-stations and is capable to utilize the GPIO's when running on a Raspberry-Pi.
 .
 *    Search and manage webradio-stations with a GUI
 *    Listen to your local MP3-Music-Collection
 *    Search your MP3-Collection
 *    Automated Album-Cover download
 *    Use of embedded Album-Covers from your MP3's
 *    support for multiple File-Formats like .mp3, .ogg, .oga, and .flac
 *    Stream Music from Youtube
 *    Check the Weather-Forecast for you location
 *    Multiple and self-designable Themes
 *    Shutdown-Timer / Sleep-Timer
 *    GPIO integration (use Buttons for changing webradio-stations, shutdown, mount-/umount USB-Sticks aso.)
 *    Onscreen settings
 *    Screensaver-Mode
Do you want to install the software package? [y/N]:y
Get:1 http://mirrordirector.raspbian.org/raspbian/ jessie/main libbinio1ldbl armhf 1.4+dfsg1-4 [22.8 kB]
Get:2 http://mirrordirector.raspbian.org/raspbian/ jessie/main libdnet armhf 2.64 [64.4 kB]
Get:3 http://mirrordirector.raspbian.org/raspbian/ jessie/main libmpcdec6 armhf 2:0.1\~r459-4.1 [32.1 kB]
Get:4 http://mirrordirector.raspbian.org/raspbian/ jessie/main mysql-common all 5.5.62-0+deb8u1 [75.7 kB]
Get:5 http://mirrordirector.raspbian.org/raspbian/ jessie/main libmysqlclient18 armhf 5.5.62-0+deb8u1 [619 kB]
Get:6 http://mirrordirector.raspbian.org/raspbian/ jessie/main libnfs4 armhf 1.9.5-2 [49.4 kB]
Get:7 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-script armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [674 kB]
Get:8 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-sql armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [122 kB]
Get:9 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-declarative armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [910 kB]
Get:10 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-designer armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [3386 kB]
Get:11 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-help armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [213 kB]
Get:12 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-scripttools armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [221 kB]
Get:13 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-sql-mysql armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [65.5 kB]
Get:14 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-svg armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [150 kB]
Get:15 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqt4-test armhf 4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1 [91.5 kB]
Get:16 http://mirrordirector.raspbian.org/raspbian/ jessie/main libqtassistantclient4 armhf 4.6.3-6 [13.5 kB]
Get:17 http://mirrordirector.raspbian.org/raspbian/ jessie/main libslp1 armhf 1.2.1-10+deb8u1 [40.4 kB]
Get:18 http://mirrordirector.raspbian.org/raspbian/ jessie/main libroar2 armhf 1.0\~beta11-1 [155 kB]
Get:19 http://mirrordirector.raspbian.org/raspbian/ jessie/main libsoxr0 armhf 0.1.1-1 [71.7 kB]
Get:20 http://mirrordirector.raspbian.org/raspbian/ jessie/main libupnp6 armhf 1:1.6.19+git20141001-1+deb8u1 [145 kB]
Get:21 http://mirrordirector.raspbian.org/raspbian/ jessie/main libzzip-0-13 armhf 0.13.62-3+deb8u1 [51.8 kB]
Get:22 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-uritemplate all 0.6-1 [4928 B]
Get:23 http://mirrordirector.raspbian.org/raspbian/ jessie/main libadplug-2.2.1-0 armhf 2.2.1+dfsg3-0.1+deb8u1 [154 kB]
Get:24 http://mirrordirector.raspbian.org/raspbian/ jessie/main libao-common armhf 1.1.0-3 [11.0 kB]
Get:25 http://mirrordirector.raspbian.org/raspbian/ jessie/main libao4 armhf 1.1.0-3 [32.1 kB]
Get:26 http://mirrordirector.raspbian.org/raspbian/ jessie/main libiso9660-8 armhf 0.83-4.2 [132 kB]
Get:27 http://mirrordirector.raspbian.org/raspbian/ jessie/main libsys-hostname-long-perl all 1.4-3 [11.6 kB]
Get:28 http://mirrordirector.raspbian.org/raspbian/ jessie/main libmail-sendmail-perl all 0.79.16-1 [26.6 kB]
Get:29 http://mirrordirector.raspbian.org/raspbian/ jessie/main libmpdclient2 armhf 2.9-1 [36.2 kB]
Get:30 http://mirrordirector.raspbian.org/raspbian/ jessie/main libresid-builder0c2a armhf 2.1.1-14 [34.7 kB]
Get:31 http://mirrordirector.raspbian.org/raspbian/ jessie/main libsidplay2 armhf 2.1.1-14 [110 kB]
Get:32 http://mirrordirector.raspbian.org/raspbian/ jessie/main libsidutils0 armhf 2.1.1-14 [25.6 kB]
Get:33 http://mirrordirector.raspbian.org/raspbian/ jessie/main mpc armhf 0.26-1 [32.6 kB]
Get:34 http://mirrordirector.raspbian.org/raspbian/ jessie/main mpd armhf 0.19.1-1.1 [305 kB]
Get:35 http://mirrordirector.raspbian.org/raspbian/ jessie/main po-debconf all 1.0.16+nmu3 [220 kB]
Get:36 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-httplib2 all 0.9+dfsg-2 [39.0 kB]
Get:37 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-lxml armhf 3.4.0-1+deb8u1 [690 kB]
Get:38 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-mpd all 0.3.0-4 [9652 B]
Get:39 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-sip armhf 4.16.4+dfsg-1 [75.0 kB]
Get:40 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-qt4 armhf 4.11.2+dfsg-1 [1661 kB]
Get:41 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-mpdclient all 0.11.1-4 [5854 B]
Fetched 10.8 MB in 6s (18.1 kB/s)                                              
Extracting templates from packages: 100%
Extracting templates from packages: 100%
Selecting previously unselected package libbinio1ldbl:armhf.
(Reading database ... 134571 files and directories currently installed.)
Preparing to unpack .../libbinio1ldbl_1.4+dfsg1-4_armhf.deb ...
Unpacking libbinio1ldbl:armhf (1.4+dfsg1-4) ...
Selecting previously unselected package libdnet:armhf.
Preparing to unpack .../libdnet_2.64_armhf.deb ...
Unpacking libdnet:armhf (2.64) ...
Selecting previously unselected package libmpcdec6:armhf.
Preparing to unpack .../libmpcdec6_2%3a0.1\~r459-4.1_armhf.deb ...
Unpacking libmpcdec6:armhf (2:0.1\~r459-4.1) ...
Selecting previously unselected package mysql-common.
Preparing to unpack .../mysql-common_5.5.62-0+deb8u1_all.deb ...
Unpacking mysql-common (5.5.62-0+deb8u1) ...
Selecting previously unselected package libmysqlclient18:armhf.
Preparing to unpack .../libmysqlclient18_5.5.62-0+deb8u1_armhf.deb ...
Unpacking libmysqlclient18:armhf (5.5.62-0+deb8u1) ...
Selecting previously unselected package libnfs4:armhf.
Preparing to unpack .../libnfs4_1.9.5-2_armhf.deb ...
Unpacking libnfs4:armhf (1.9.5-2) ...
Selecting previously unselected package libqt4-script:armhf.
Preparing to unpack .../libqt4-script_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-script:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-sql:armhf.
Preparing to unpack .../libqt4-sql_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-sql:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-declarative:armhf.
Preparing to unpack .../libqt4-declarative_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-declarative:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-designer:armhf.
Preparing to unpack .../libqt4-designer_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-designer:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-help:armhf.
Preparing to unpack .../libqt4-help_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-help:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-scripttools:armhf.
Preparing to unpack .../libqt4-scripttools_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-scripttools:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-sql-mysql:armhf.
Preparing to unpack .../libqt4-sql-mysql_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-sql-mysql:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-svg:armhf.
Preparing to unpack .../libqt4-svg_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-svg:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqt4-test:armhf.
Preparing to unpack .../libqt4-test_4%3a4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1_armhf.deb ...
Unpacking libqt4-test:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Selecting previously unselected package libqtassistantclient4:armhf.
Preparing to unpack .../libqtassistantclient4_4.6.3-6_armhf.deb ...
Unpacking libqtassistantclient4:armhf (4.6.3-6) ...
Selecting previously unselected package libslp1:armhf.
Preparing to unpack .../libslp1_1.2.1-10+deb8u1_armhf.deb ...
Unpacking libslp1:armhf (1.2.1-10+deb8u1) ...
Selecting previously unselected package libroar2.
Preparing to unpack .../libroar2_1.0\~beta11-1_armhf.deb ...
Unpacking libroar2 (1.0\~beta11-1) ...
Selecting previously unselected package libsoxr0:armhf.
Preparing to unpack .../libsoxr0_0.1.1-1_armhf.deb ...
Unpacking libsoxr0:armhf (0.1.1-1) ...
Selecting previously unselected package libupnp6.
Preparing to unpack .../libupnp6_1%3a1.6.19+git20141001-1+deb8u1_armhf.deb ...
Unpacking libupnp6 (1:1.6.19+git20141001-1+deb8u1) ...
Selecting previously unselected package libzzip-0-13:armhf.
Preparing to unpack .../libzzip-0-13_0.13.62-3+deb8u1_armhf.deb ...
Unpacking libzzip-0-13:armhf (0.13.62-3+deb8u1) ...
Selecting previously unselected package python-uritemplate.
Preparing to unpack .../python-uritemplate_0.6-1_all.deb ...
Unpacking python-uritemplate (0.6-1) ...
Selecting previously unselected package libadplug-2.2.1-0.
Preparing to unpack .../libadplug-2.2.1-0_2.2.1+dfsg3-0.1+deb8u1_armhf.deb ...
Unpacking libadplug-2.2.1-0 (2.2.1+dfsg3-0.1+deb8u1) ...
Selecting previously unselected package libao-common.
Preparing to unpack .../libao-common_1.1.0-3_armhf.deb ...
Unpacking libao-common (1.1.0-3) ...
Selecting previously unselected package libao4.
Preparing to unpack .../libao4_1.1.0-3_armhf.deb ...
Unpacking libao4 (1.1.0-3) ...
Selecting previously unselected package libiso9660-8.
Preparing to unpack .../libiso9660-8_0.83-4.2_armhf.deb ...
Unpacking libiso9660-8 (0.83-4.2) ...
Selecting previously unselected package libsys-hostname-long-perl.
Preparing to unpack .../libsys-hostname-long-perl_1.4-3_all.deb ...
Unpacking libsys-hostname-long-perl (1.4-3) ...
Selecting previously unselected package libmail-sendmail-perl.
Preparing to unpack .../libmail-sendmail-perl_0.79.16-1_all.deb ...
Unpacking libmail-sendmail-perl (0.79.16-1) ...
Selecting previously unselected package libmpdclient2.
Preparing to unpack .../libmpdclient2_2.9-1_armhf.deb ...
Unpacking libmpdclient2 (2.9-1) ...
Selecting previously unselected package libresid-builder0c2a.
Preparing to unpack .../libresid-builder0c2a_2.1.1-14_armhf.deb ...
Unpacking libresid-builder0c2a (2.1.1-14) ...
Selecting previously unselected package libsidplay2.
Preparing to unpack .../libsidplay2_2.1.1-14_armhf.deb ...
Unpacking libsidplay2 (2.1.1-14) ...
Selecting previously unselected package libsidutils0.
Preparing to unpack .../libsidutils0_2.1.1-14_armhf.deb ...
Unpacking libsidutils0 (2.1.1-14) ...
Selecting previously unselected package mpc.
Preparing to unpack .../archives/mpc_0.26-1_armhf.deb ...
Unpacking mpc (0.26-1) ...
Selecting previously unselected package mpd.
Preparing to unpack .../mpd_0.19.1-1.1_armhf.deb ...
Unpacking mpd (0.19.1-1.1) ...
Selecting previously unselected package po-debconf.
Preparing to unpack .../po-debconf_1.0.16+nmu3_all.deb ...
Unpacking po-debconf (1.0.16+nmu3) ...
Selecting previously unselected package python-httplib2.
Preparing to unpack .../python-httplib2_0.9+dfsg-2_all.deb ...
Unpacking python-httplib2 (0.9+dfsg-2) ...
Selecting previously unselected package python-lxml.
Preparing to unpack .../python-lxml_3.4.0-1+deb8u1_armhf.deb ...
Unpacking python-lxml (3.4.0-1+deb8u1) ...
Selecting previously unselected package python-mpd.
Preparing to unpack .../python-mpd_0.3.0-4_all.deb ...
Unpacking python-mpd (0.3.0-4) ...
Selecting previously unselected package python-sip.
Preparing to unpack .../python-sip_4.16.4+dfsg-1_armhf.deb ...
Unpacking python-sip (4.16.4+dfsg-1) ...
Selecting previously unselected package python-qt4.
Preparing to unpack .../python-qt4_4.11.2+dfsg-1_armhf.deb ...
Unpacking python-qt4 (4.11.2+dfsg-1) ...
Selecting previously unselected package python-mpdclient.
Preparing to unpack .../python-mpdclient_0.11.1-4_all.deb ...
Unpacking python-mpdclient (0.11.1-4) ...
Processing triggers for man-db (2.7.5-1\~bpo8+1) ...
Processing triggers for systemd (215-17+deb8u13) ...
Setting up libbinio1ldbl:armhf (1.4+dfsg1-4) ...
Setting up libdnet:armhf (2.64) ...
Setting up libmpcdec6:armhf (2:0.1\~r459-4.1) ...
Setting up mysql-common (5.5.62-0+deb8u1) ...
Setting up libmysqlclient18:armhf (5.5.62-0+deb8u1) ...
Setting up libnfs4:armhf (1.9.5-2) ...
Setting up libqt4-script:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-sql:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-declarative:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-designer:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-help:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-scripttools:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-sql-mysql:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-svg:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqt4-test:armhf (4:4.8.6+git64-g5dc8b2b+dfsg-3+deb8u2+rpi1) ...
Setting up libqtassistantclient4:armhf (4.6.3-6) ...
Setting up libslp1:armhf (1.2.1-10+deb8u1) ...
Setting up libroar2 (1.0~beta11-1) ...
Setting up libsoxr0:armhf (0.1.1-1) ...
Setting up libupnp6 (1:1.6.19+git20141001-1+deb8u1) ...
Setting up libzzip-0-13:armhf (0.13.62-3+deb8u1) ...
Setting up python-uritemplate (0.6-1) ...
Setting up libadplug-2.2.1-0 (2.2.1+dfsg3-0.1+deb8u1) ...
Setting up libao-common (1.1.0-3) ...
Setting up libao4 (1.1.0-3) ...
Setting up libiso9660-8 (0.83-4.2) ...
Setting up libsys-hostname-long-perl (1.4-3) ...
Setting up libmail-sendmail-perl (0.79.16-1) ...
Setting up libmpdclient2 (2.9-1) ...
Setting up libresid-builder0c2a (2.1.1-14) ...
Setting up libsidplay2 (2.1.1-14) ...
Setting up libsidutils0 (2.1.1-14) ...
Setting up mpc (0.26-1) ...
Setting up mpd (0.19.1-1.1) ...
Setting up po-debconf (1.0.16+nmu3) ...
Setting up python-httplib2 (0.9+dfsg-2) ...
Setting up python-lxml (3.4.0-1+deb8u1) ...
Setting up python-mpd (0.3.0-4) ...
Setting up python-sip (4.16.4+dfsg-1) ...
Setting up python-qt4 (4.11.2+dfsg-1) ...
Setting up python-mpdclient (0.11.1-4) ...
Processing triggers for libc-bin (2.19-18+deb8u10) ...
Processing triggers for systemd (215-17+deb8u13) ...
Selecting previously unselected package webradio-py.
(Reading database ... 135159 files and directories currently installed.)
Preparing to unpack webradio-py_0.3.7_armhf.deb ...
Unpacking webradio-py (0.3.7) ...
Setting up webradio-py (0.3.7) ...
Install eyeD3 to be able to preview embedded IDv3 Album-Arts
Downloading/unpacking eyeD3
  Downloading eyeD3-0.8.10-py2.py3-none-any.whl (147kB): 147kB downloaded
Cleaning up...
Exception:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python2.7/dist-packages/pip/commands/install.py", line 290, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/usr/lib/python2.7/dist-packages/pip/req.py", line 1266, in prepare_files
    req_to_install.extras):
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2401, in requires
    dm = self._dep_map
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2597, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2621, in _compute_dependencies
    parsed = next(parse_requirements(distvers))
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 49, in <lambda>
    next = lambda o: o.next()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2721, in parse_requirements
    "version spec")
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2686, in scan_list
    raise ValueError(msg, line, "at", line[p:])
ValueError: ('Expected version spec in', 'pylast ~=2.0', 'at', ' ~=2.0')

Storing debug log for failure in /root/.pip/pip.log
dpkg: error processing package webradio-py (--install):
 subprocess installed post-installation script returned error exit status 2
Processing triggers for gnome-menus (3.13.3-6) ...
Processing triggers for desktop-file-utils (0.22-1) ...
Processing triggers for mime-support (3.58) ...
Processing triggers for hicolor-icon-theme (0.13-1) ...
Errors were encountered while processing:
 webradio-py`

The pip.log is (tail):

Downloading from URL https://files.pythonhosted.org/packages/75/66/ b1d97f06499199b3b3dfcd25e3f000c4256e7f313b3a0aae2e3d24e6acb7/ eyeD3-0.8.10-py2.py3-none-ny.whl#sha256=3e10b7272bfe94948c42ccf4137fc72e333dd03b84b756b3fb646be1dcad9f8d (from https://pypi.org/simple/eyed3/)
Cleaning up...
Exception:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python2.7/dist-packages/pip/commands/install.py", line 290, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/usr/lib/python2.7/dist-packages/pip/req.py", line 1266, in prepare_files req_to_install.extras):
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2401, in requires  dm = self._dep_map
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2597, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2621, in _compute_dependencies
    parsed = next(parse_requirements(distvers))
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 49, in <lambda>
    next = lambda o: o.next()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2721, in parse_requirements
    "version spec")
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2686, in scan_list
    raise ValueError(msg, line, "at", line[p:])
ValueError: ('Expected version spec in', 'pylast ~=2.0', 'at', ' ~=2.0')

**Comment by Acer54** [comment]
Hi John,
I've never tried to install the deb pack with gdebi to be honest. Can you try the installation with the suggested command [from the wiki-guideline ](https://github.com/Acer54/Webradio_v2/wiki/3_Installation-webradio-py) `sudo apt install ./webradio-py_0.3.7_armhf.deb`?
If the error still remains, I would try to exclude the installation of eyed3 in first step and install it later on with pip.
I am not sure where the problem is, maybe with different versions of python or pip in raspbian jessy... I've testet the deb in stretch and buster so far...

**Comment by braselectron** [comment]
Zunächst vielen Dank für die prompte Rückkehr zu meiner Frage.

I will reinstall following your suggestion.  I did use gdebi because it is better for debugging and fixing dependence problems.

I will post answer here in a few moments, just need to return to my SOHO.

danke im Voraus / Obrigado.

**Comment by braselectron** [comment]
If I just start with:
sudo apt install ./web.....deb
I get many errors.

WARNING: apt does not have a stable CLI interface yet. Use with caution in scrip
ts.

Reading package lists...
Building dependency tree...
Reading state information...
E: Release 'webradio-py_0.3.7_armhf.deb' for '0ad' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for '0ad-data' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for '0ad-data-common' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'libboost-filesystem1.55.0' was not
 found
.... so on and on....
E: Release 'webradio-py_0.3.7_armhf.deb' for 'pixel-wallpaper' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'rpd-wallpaper' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'point-rpi' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'pprompt' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'raspberrypi-ui-mods' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'rc-gui' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'kweb' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'kweb3' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'omxplayergui.py' was not found
E: Release 'webradio-py_0.3.7_armhf.deb' for 'webradio-py' was not found

When I try to purge/remove webradio-py with:
sudo apt-get --purge remove webradio-py
or
sudo apt-get --purge remove mpd
or
sudo apt-get --reinstall install mpd
or
sudo apt-get --reinstall install ./webradio.....deb

I get this error:
The following packages will be REMOVED:
  webradio-py
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Do you want to continue? [Y/n] Y
(Reading database ... 135159 files and directories currently installed.)
Removing webradio-py (0.3.7) ...
rm: cannot remove ‘/etc/mpd.conf’: No such file or directory
dpkg: error processing package webradio-py (--remove):
 subprocess installed post-removal script returned error exit status 1
Errors were encountered while processing:
 webradio-py
E: Sub-process /usr/bin/dpkg returned an error code (1)

Even with:
sudo dpkg --purge --force-remove-reinstreq webradio-py
I get:
(Reading database ... 135131 files and directories currently installed.)
Removing webradio-py (0.3.7) ...
rm: cannot remove ‘/etc/mpd.conf’: No such file or directory
dpkg: error processing package webradio-py (--purge):
 subprocess installed post-removal script returned error exit status 1
Errors were encountered while processing:
 webradio-py

I'm stuck!

**Comment by braselectron** [comment]
**uname -a**
Linux rasp109 4.9.35-v7+ #1014 SMP Fri Jun 30 14:47:43 BST 2017 armv7l GNU/Linux

**lsb_release -a**
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 8.0 (jessie)
Release:	8.0
Codename:	jessie

**pip --version**
pip 1.5.6 from /usr/lib/python2.7/dist-packages (python 2.7)
**pip3 --version**
pip 1.5.6 from /usr/lib/python3/dist-packages (python 3.4)
**python --version**
Python 2.7.9
**python3 --version**
Python 3.4.2

**Comment by braselectron** [comment]
UPDATE:

I did this:
cd /var/lib/dpkg/inf
then
mkdir ~/temp
then  (remove left over of bad install - to play safe move to ~/temp)
ls webradio* | while read item; do sudo mv $item /home/pi/temp; done
then
sudo apt-get install -f
>>clean<<
then
sudo apt install mpd
then
sudo apt-get install -f
then
sudo apt-get autoremove
then
sudo apt-get install --reinstall youtube-dl

Now "sudo apt update" is clean again.
I then returned to ~/Download folder and repeated the command:
sudo apt install ./webradio.....deb
lots of messages about not found packages: 
E: Release 'webradio-py_0.3.7_armhf.deb' for '<installed packages>' was not found

So I will try to download 0.3.5 and post here result.

**Comment by Acer54** [comment]
Thats really strange. Never have seen this kind of failures until now...

But you are using jessie... (never tested my selfmade-deb-package with jessie, only with stretch and buster...) 

Maybe jessie needs to use "apt-get" instead of "apt" .
But the main problem still remains: You cannot install "eyeD3" (even not manually). 
Have you tried to install (apt-get or gdebi) WITHOUT eyeD3 like I was mentioning in my first reply?

**Comment by Acer54** [comment]
Hi John, today I took the time to setup a fresh installation of Raspbian Jessie, making sure that I am using the same setup than you do.

In fact, I can reproduce the problem. I also got the same error message when trying to install "eyeD3".

I found that this is caused by an old "pip" version which comes with jessie...

**SOLUTION:**
Before installing the deb-package, upgrade your pip version with 

> sudo pip install -U pip && pip --version
pip 19.2.1 from /usr/local/lib/python2.7/dist-packages/pip (python 2.7)

After this, install the deb-package with

> sudo dpkg -i webradio-py_0.3.7_armhf.deb 

This approach will not customize your installation unfortunatelly, but will take the "default-values" instead.
Therefor you can customize it separately after a successful installation with

> sudo dpkg-reconfigure webradio-py 

Hope this is helpful to you!

Regards,
Matthias

**Comment by braselectron** [comment]
Dear Matthias,

Thank you for your promptly and detailed feedback.  I am very glad that I was getting here to the same conclusion.  PIP always surprises me from time to time.  I have had many issues in many projects, but I really like python.  I hope to be able to test the solution this weekend.

I did learn alot these last few days testing other home-brewed solutions and I believe I found another bug in mpd/mpc, related to id3tagging when using "lame", and also a performance issues with vorbis while I was testing the packages here.  I will open ticket at the mpd project about that.

I would like to thank your resilience in keeping great projects up and running, and that this is very important for students and developer to build new skills and for me lots of fun with troubleshooting  - great mind puzzles - at the end we all win a fantastic new package/product.
